agent/pkg/dns/module.go

```diff
@@ -9,14 +9,14 @@ import (
 	"github.com/kubeedge/edgemesh/agent/pkg/dns/controller"
 	"github.com/kubeedge/edgemesh/common/informers"
 	"github.com/kubeedge/edgemesh/common/modules"
-	"github.com/kubeedge/edgemesh/common/util"
 )
 
+const InterfaceAddress = "172.17.0.1/24"
```

Comment:
Is the address of the dummy interface 169.254.*.* better?

Reply:
Maybe it would be better if we make it user-configurable, like:
```
apiVersion: agent.edgemesh.config.kubeedge.io/v1alpha1
kind: EdgeMeshAgent
commonConfig:
  dummyIfName: "edgemesh-dummy0"
  dummyIfCIDR: "169.254.0.0/16"
kubeAPIConfig:
......
```

Reply:
I am free for it, but keep doubt about this value.

Reply:
OK, I think we can discuss this design at the KubeEdge community meeting.